core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/machineLearning/Scorer/MachineLearningScorerOpDesc.scala

```diff
@@ -0,0 +1,179 @@
+  @JsonSchemaInject(
+    strings = Array(
+      new JsonSchemaString(path = HideAnnotation.hideTarget, value = "isRegression"),
+      new JsonSchemaString(path = HideAnnotation.hideType, value = HideAnnotation.Type.equals),
+      new JsonSchemaString(path = HideAnnotation.hideExpectedValue, value = "true")
+    )
+  )
+  var classificationMetrics: List[classificationMetricsFnc] = List()
+
+  @JsonProperty(required = false, value = "regressionFlag")
+  @JsonSchemaTitle("Scorer Functions")
+  @JsonPropertyDescription("Select regression tasks metrics")
+  @JsonSchemaInject(
+    strings = Array(
+      new JsonSchemaString(path = HideAnnotation.hideTarget, value = "isRegression"),
+      new JsonSchemaString(path = HideAnnotation.hideType, value = HideAnnotation.Type.equals),
+      new JsonSchemaString(path = HideAnnotation.hideExpectedValue, value = "false")
+    )
+  )
+  var regressionMetrics: List[regressionMetricsFnc] = List()
+
+  override def operatorInfo: OperatorInfo =
+    OperatorInfo(
+      "Machine Learning Scorer",
+      "Scorer for machine learning models",
+      OperatorGroupConstants.MACHINE_LEARNING_GENERAL_GROUP,
+      inputPorts = List(InputPort()),
+      outputPorts = List(OutputPort())
+    )
+  override def getOutputSchema(schemas: Array[Schema]): Schema = {
+    val outputSchemaBuilder = Schema.builder()
+    if (!isRegression) {
+      outputSchemaBuilder.add(new Attribute("Class", AttributeType.STRING))
+    }
+
+    val metrics = if (isRegression) {
+      regressionMetrics.map(_.getName())
+    } else {
+      classificationMetrics.map(_.getName())
+    }
+    metrics.foreach(metricName => {
+      outputSchemaBuilder.add(new Attribute(metricName, AttributeType.DOUBLE))
+    })
+
+    outputSchemaBuilder.build()
+  }
+
+  private def getClassificationScorerName(scorer: classificationMetricsFnc): String = {
+    // Directly return the name of the scorer using the getName() method
+    scorer.getName()
+  }
+  private def getRegressionScorerName(scorer: regressionMetricsFnc): String = {
+    // Directly return the name of the scorer using the getName() method
+    scorer.getName()
+  }
+
+  private def getEachScorerName(scorer: Any): String =
+    scorer match {
+      case s: classificationMetricsFnc => getClassificationScorerName(s)
+      case s: regressionMetricsFnc     => getRegressionScorerName(s)
+      case _                           => throw new IllegalArgumentException("Unknown scorer type")
+    }
+
+  private def getSelectedScorers(): String = {
+    // Return a string of scorers using the getEachScorerName() method
+    var scorers: List[_] = List()
+    if (isRegression) scorers = regressionMetrics
+    else scorers = classificationMetrics
+
+    scorers.map(scorer => getEachScorerName(scorer)).mkString("'", "','", "'")
+  }
+
+  override def generatePythonCode(): String = {
+    var is_regression = "False"
+    if (isRegression) is_regression = "True"
```

Comment:
```suggestion
    val isRegressionStr = if (isRegression) "False" else "True"
```

Reply:
Fix.
Thank you for reviewing my pull request.